modules/auxiliary/gather/impersonate_ssl.rb

```diff
@@ -32,7 +32,7 @@ def initialize(info = {})
     register_options(
       [
         Opt::RPORT(443),
-        OptString.new('SNI', [false, 'Server Name Indicator', nil]),
+        OptString.new('SSLServerNameIndication', [ false, 'SSL/TLS Server Name Indication (SNI)', nil]),
```

Comment:
I think we want to use the `aliases:` kwarg here to retain backwards compat.
```suggestion
        OptString.new('SSLServerNameIndication', [ false, 'SSL/TLS Server Name Indication (SNI)', nil], aliases: ['SNI']),
```

Reply:
Oops, thanks! I had an alias added in the initial commit, but I wanted to align the wording across modules - and it looks like I dropped the alias in the copy/paste 😄